README.md

```diff
@@ -1,83 +1,5 @@
 # ChaosBot
 
-> Chaos, the vacant and infinite space which existed according to the ancient
-> cosmogonies previous to the creation of the world, and out of which the gods,
-> men, and all things arose.
-
-ChaosBot is a social coding experiment to see what happens when the absolute
-direction of a software project is turned over to the open source community.
+### _CHAAAAAAAAAAAOOOOOOOOOSSSSSSS_
```

Comment:
Or you know, should be italics and all caps twice. Like, twice as upper case characters.